en/api_reference/classdronecore_1_1_device.md

```diff
@@ -81,15 +81,15 @@ uint8_t | [get_own_component_id](#classdronecore_1_1_device_1aa6dc1543b0ae9439a6
 ## Constructor & Destructor Documentation
 
 
-### Device() {#classdronecore_1_1_device_1af07b4defa38736ce928377af2f31fb6e}
+### Device() {#classdronecore_1_1_device_1a507d542a9a948033f2873dd17156fdc7}
 ```cpp
-dronecore::Device::Device(DroneCoreImpl *parent, uint8_t target_system_id)
+dronecore::Device::Device(DroneCoreImpl &parent, uint8_t target_system_id)
 ```
 
 
 **Parameters**
 
-* DroneCoreImpl * **parent** - 
+* DroneCoreImpl & **parent** - 
```

Comment:
Is there a space between `&` and **parent** ?
I think `DroneCoreImpl &parent` looks better than `DroneCoreImpl & parent`. What do you think ?

Reply:
This is what is generated by the toolchain (not this specific change), so IMO we should accept this PR and deal with the rendering as a separate issue.

I'll merge and lets carry on the conversation in https://github.com/dronecore/DroneCore/issues/321